_data/toc/testing.yml

```diff
@@ -40,9 +40,6 @@ pages:
         - label: Jasmine
           url: /test/js/jasmine.html
 
-        - label: JsTestDriver
```

Comment:
If this is still available in 2.1, we should still keep the link in the TOC for that specific version.
It would probably look like the following:
``` yml
- label: JsTestDriver
  versions: ["2.1"]
  url: /test/js/test_js-unit.html
```

Reply:
Of course!